spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe User, js: true do
```

Comment:
Do we need js on for this one?

Reply:
(Does it matter?)

Reply:
I don't think it matters I can test to see if theres any performance difference

Reply:
Looks like we always need to have `js: true` for capybara otherwise I get weird errors